src/components/ModalFooter/__stories__/ModalFooter.stories.mdx

```diff
@@ -0,0 +1,144 @@
+import ModalFooter from "../ModalFooter";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate, createStoryMetaSettings } from "../../../storybook/functions/create-component-story";
+
+export const metaSettings = createStoryMetaSettings({
+  component: ModalFooter,
+  enumPropNamesArray: [], // List enum props here
+  iconPropNamesArray: [], // List props that are typed as icons here
+  actionPropsArray: [], // List the component's actions here
+});
+
+<Meta
+  title="Feedback/Modal/ModalFooter"
+  component={ ModalFooter }
+  argTypes={ metaSettings.argTypes }
+  decorators={ metaSettings.decorators }
+/>
+
+<!--- Component template -->
+
+export const modalFooterTemplate = createComponentTemplate(ModalFooter);
+
+<!--- Component documentation -->
+
+# ModalFooter
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Please write here the component description
+
+<Canvas>
+  <Story name="Overview"
+         args={ {prop1: "prop's default value", prop2: "prop's default value"} }>
+    { modalFooterTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ ModalFooter } />
+
+## Usage
+<UsageGuidelines guidelines={[
```

Comment:
We want to create  story for each modal building block? If we do let's remove all plop template examples (such as empty stories and generic "Usage guideline 1")
If not let's delete it

Reply:
👍